### ISSUE TITLE
Master Offers component v2

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#3463]
Great idea; I guess we can make a functional component with an address as param and input and merge these three current ones. 

When the logged-in user visits /bsx/offers it lands on its own address stats by default, i.e. in my case 
`/bsx/offers?u=bXgvP2cQzEnPvuPQHRghz9dUYovqzyPhXBRXbNwVCef7nsmqb`
where default would be incoming offers

There should be two filters where you can choose
- F1
  - created offers by particular address
  - incoming offers by particular address
- F2 
  - Active
  - Accepted
  - Withdrawn

When the non-logged user opens it, they should see a global table like at https://beta.kodadot.xyz/bsx/stats, so F1 should be disabled.

The goal would be for anyone/user can see per address offers 
- created
- incoming 
- global 

We can put it under the Stats dropdown in the navbar with the name Offers. 

Later it becomes the replacement for current stats

Work can start once the issues below are completed. 
- #3450 
- #3203 
- #2801

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/3462#issuecomment-1187044108_

**Comment by prachi00** [comment]
👋

**Comment by kodabot** [comment]
ISSUE BLOCKED - issue #3463 is blocked. Please wait until issue is unblocked.

**Comment by prachi00** [comment]
I can work on this once unblocked 😊

**Comment by prachi00** [comment]
> Adding comment here to follow from
> 
> * [#3203 add table to show currently standing offers for user #3477](https://github.com/kodadot/nft-gallery/pull/3477)
> 
> It would be nice to have some sort of option (maybe toggle) to show only active offers, since those are the ones which will be (I assume) most interesting to the user most of the time

I guess we can add filters like the history table?

**Comment by petersopko** [comment]
> > Adding comment here to follow from
> > 
> > * [#3203 add table to show currently standing offers for user #3477](https://github.com/kodadot/nft-gallery/pull/3477)
> > 
> > It would be nice to have some sort of option (maybe toggle) to show only active offers, since those are the ones which will be (I assume) most interesting to the user most of the time
> 
> I guess we can add filters like the history table?

yup, I guess that would be the best way to go, something like this:
![image](https://user-images.githubusercontent.com/73316633/180165368-c8c63fa6-ed33-4ac7-8d0d-383ac76cc50a.png)

**Comment by prachi00** [comment]
@yangwao how are we keeping the UI for this one? do we have any reference? 
like are we giving an option here like Transfer, a dedicated page to check offer for any address? 

<img width="350" alt="Screen Shot 2022-07-22 at 4 24 40 PM" src="https://user-images.githubusercontent.com/39299315/180580126-1355316f-72cf-44f9-b3b6-789641c480f9.png">

**Comment by prachi00** [comment]
> @yangwao how are we keeping the UI for this one? do we have any reference? like are we giving an option here like Transfer, a dedicated page to check offer for any address?
> 
> <img alt="Screen Shot 2022-07-22 at 4 24 40 PM" width="350" src="https://user-images.githubusercontent.com/39299315/180580126-1355316f-72cf-44f9-b3b6-789641c480f9.png">
@yangwao need a little help with the ui ux part of this oneee

**Comment by yangwao** [comment]
> a dedicated page to check offer for any address?

yes.
check original issue, I've just updated it :)

**Comment by prachi00** [comment]
Okay I'll start working on this one

**Comment by prachi00** [comment]
@yangwao so when you say F1 filter, do you mean two tabs? where one tab shows
created offers by particular address TABLE
and other shows
incoming offers by particular address TABLE
and at the top we can have a text input field where we can input the particular address?

**Comment by kodabot** [comment]
ASSIGNED - @prachi00 🔒 LOCKED -> Monday, August 8th 2022, 05:13:56 UTC -> 72 hours
